omero/sysadmins/version-requirements.txt

```diff
@@ -149,7 +149,7 @@ The planned changes for 5.2 depend upon our support of
   * [5.2] 3.4 dropped, 3.5 deprecated, 3.6 not supported
```

Comment:
good point. We also need to discuss if 3.4 is dropped cf. https://trello.com/c/QvS3yU4h/95-blocker-revert-requirement-changes